jobs/build/ocp4/build.groovy

```diff
@@ -366,6 +366,19 @@ def stageBuildImages() {
 
         buildlib.doozer "${doozerOpts} images:streams mirror"
     }
+    // check new embargoed build
+    out = commonlib.shell(
+        returnAll: true,
+        script: "${env.WORKSPACE}/build-scripts/querybuilds.py -v ${version.stream} -t ${version.branch} -i ${buildPlan.imagesIncluded}"
```

Comment:
In stead of inventing a new script, could you have a look at `doozer detect embargo` and see if it applicable?

Reply:
thanks for the info, looks like doozer detect-embargo don't support images list as args ? and for this task I think it just need a fast and light method to do a simple check in ocp4 job.

Reply:
The problem is that `.p1` in the release field only means "the source of the image is embargoed". An image should also be considered embargoed if it contains embargoed rpms (".p1" or tagged with `-embargoed`).
This is somewhat complicated. So please don't duplicate the logic here.

Run `doozer detect-embargo` for help. If the existing Doozer command is not powerful enough for this use case, you can extend it.

Reply:
I am not sure what you mean by `images list as args` here. doozer detect-embargo supports detecting by nvrs or release payloads.